spring-xsuaa/pom.xml

```diff
@@ -49,6 +50,11 @@
 			<artifactId>caffeine</artifactId>
 			<version>${caffeine.version}</version>
 		</dependency>
+		<dependency>
```

Comment:
This library should not bring projectreactor as a transitive dependency. There are projects using this library who don't have reactor in their classpath and don't need it. So this dependency should be `optional=true` (I guess, it could also be `scope=provided`).